introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-transfer-learning-highlevel.ipynb

```diff
@@ -54,8 +54,11 @@
     "print(role)\n",
```

Comment:
<p>Is there a better way of specifying data bucket name? First, pointing to alpha doesn't seem right. Second, is it going to work in non-us-west-2 regions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Regions: From what I understand, bucket will be copied to all regions and that is the premise under which other penny related work is being done. </p><p> </p><p>bucket_name: you mean changing the name "data_bucket", "data_prefix"? Any suggestions?</p>

Reply:
<p><code>data_bucket</code>  is set to <code>penny-cache-alpha-us-west-2</code>.  The name of the variable is fine. My concern is about the value. When the notebook is replicated world-wide, will it still point to the alpha bucket?</p>

Reply:
<p>Checking with Jonathan about this. </p>

Reply:
What was the resolution on the bucket name?

Reply:
<p>There would be a bucket with name jump-start .... and it would be replicated in all regions. Will update it soon.</p>